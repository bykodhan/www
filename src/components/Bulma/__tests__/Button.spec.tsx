import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Button';

describe('<Button />', () => {
  test('should render correctly', () => {
    const tree = renderer
      .create(
        <Button
          element="a"
          anchor={{ href: 'https://deepintr.org' }}
          title="Deepin TÃ¼rkiye"
          rounded
          size="medium"
          color="success"
        >
          deepintr.org
        </Button>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
