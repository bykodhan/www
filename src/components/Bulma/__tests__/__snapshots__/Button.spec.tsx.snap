// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render correctly 1`] = `
<a
  aria-label="Deepin Türkiye"
  className="button is-success is-rounded is-size"
  href="https://deepintr.org"
  title="Deepin Türkiye"
>
  deepintr.org
</a>
`;
